index.html

```diff
@@ -6,17 +6,19 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
   <script src="https://kit.fontawesome.com/89d87f2112.js" crossorigin="anonymous"></script>
   <title>Portafolio</title>
 </head>
 
 <body>
     <header class="header">
+      <div class="crossed"></div>
       <div class="tallbar"></div>
       <div class="header-sub">
         <a href="" class="aa"><h2 class="logo">My logo</h2></a>
-        <nav class="bb">
-          <ul class="nav-elements">
+        <nav>
+          <ul class="nav-elements bb">
             <li><a href="#work">portifolio</a></li>
             <li><a href="#about me">About</a></li>
             <li><a href="#contact-page">Contact</a></li>
```

Comment:
- [ ] I recommend following the project's requirements by matching the Figma design for the mobile menu. Currently, you only have two steps left to achieve that:
1. When the mobile menu appears the content in the background should be blurred.
2.  The mobile menu should fill the whole page when it appears.
<table>
<tr>
<th>Your design</th>
<th>Figma</th>
</tr>
  <tr>
    <td><img src="https://github.com/tatouroberto/portfolio-project/assets/86069740/0aa8f010-2bb3-4013-a5c5-bfb7cb83e11f"/></td>
    <td><img src="https://github.com/tatouroberto/portfolio-project/assets/86069740/0d55db51-0bad-4812-8029-4fe224e7b7fd"/></td>
  </tr>
</table>